src/libtorch.cc

```diff
@@ -848,6 +917,10 @@ ModelInstanceState::Execute(
     // enable/disable optimized execution
     torch::jit::setGraphExecutorOptimize(
         model_state_->EnabledOptimizedExecution());
+    torch::jit::setTensorExprFuserEnabled(
+        model_state_->EnabledTensorExprFuser());
```

Comment:
> So I think this fuser logic needs to be combined with the nvfuser_pair logic below so that is fuser is false both fuser and nvfuser is disabled, otherwise use the existing logic below. @CoderHam what do you think?

Yes. It does need to be combined to ensure consistent behaviour. 

@ryanleary why do we need ENABLE_TENSOR_FUSER as well as ENABLE_NVFUSER?